HotSpot/Models/Booking.m

```diff
@@ -106,17 +107,17 @@ + (void)getCurrentBookingsWithBlock:(void(^)(NSArray<Booking *> *bookings, NSErr
     PFRelation *relation = [[PFUser currentUser] relationForKey:@"bookings"];
     PFQuery *query = relation.query;
     [query orderByAscending:@"startTime"];
-    [query whereKey:@"startTime" greaterThan:[[NSDate alloc] init]];
+    [query whereKey:@"startTime" greaterThan:[NSDate date]];
```

Comment:
i'm not sure why you have 2 queries going on here. could we please chat about this?